sos/report/plugins/openstack_nova.py

```diff
@@ -29,6 +29,7 @@ class OpenStackNova(Plugin):
     var_puppet_gen = "/var/lib/config-data/puppet-generated/nova"
     service_name = "openstack-nova-api.service"
     apachepkg = None
+    postproc_dirs = ["/etc/nova/",]
```

Comment:
do we need to comma here, I think we only do that for tuples where it makes sense?

Reply:
Indeed, it is obsolete here for the sake of iterating through the list. I added it there just in case we expand the list, to prevent implicit concatenation of adjacent string items.